modules/core/shared/src/main/scala/gem/GuideEnvironment.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+
+import cats._
+import cats.implicits._
+
+sealed trait GuideEnvironment extends Product with Serializable
+
+object GuideEnvironment {
+
+  final case class Flamingos2(options: GuideOptions[GuideGroup.Flamingos2]) extends GuideEnvironment
```

Comment:
I wonder if we should use the same class name as for `Instrument`. Perhaps `FlamingosGE`?

Reply:
Should `options` be specified on the `trait`?

Reply:
> Should `options` be specified on the trait?

The point of the `GuideEnvironment` is to eat the type parameter so it isn't clear to me what the type of `options` would be if I include it on the `trait`.  I suppose we could widen the individual `options` to `GuideOptions[GuideGroup]` though if it becomes useful?  Something like:

```scala
sealed trait GuideEnvironment extends Product with Serializable {
  def options: GuideOptions[GuideGroup]
}

object GuideEnvironment {
  final case class Flamingos2(flamingos2Options: GuideOptions[GuideGroup.Flamingos2]) extends GuideEnvironment {
    override def options: GuideOptions[GuideGroup] =
      flamingos2Options.widen[GuideGroup]
  ...
}
```